system/lib/fetch/wasmfs.cpp

```diff
@@ -0,0 +1,55 @@
+// // Copyright 2021 The Emscripten Authors.  All rights reserved.
+// // Emscripten is available under two separate licenses, the MIT license and the
+// // University of Illinois/NCSA Open Source License.  Both these licenses can be
+// // found in the LICENSE file.
```

Comment:
It would probably be good to create `system/lib/wasmfs/wasmfs.cpp` rather than putting this under `system/lib/fetch`.

It would also be good to add a file-level comment saying what this file is for and what its implementation status is.